### DISCUSSION TITLE
如果能集成mermaid就太好了，但是好像显示不出来，换了操作系统，换了浏览器都不行

### DISCUSSION THREAD

**Discussion by iaiuse** [doocs/md#415] [General]
我的文章里面经常会用到mermaid的内容，如果能转换成图片就好了，图床可以用我自己的。最好是直接能放到公众号里面的图片，那就最好了。昨天测试了一下，mermai会变成图标
![image](https://github.com/user-attachments/assets/2d5c56ea-05cd-4f3a-aed4-5bb23c9ec15e)

**Comment by YangFong** [comment]
不好意思，这个问题我们也发现了，但是还未找到合适的方式修复。

我们提供了一个临时方案：https://doocs-md.pages.dev

与原站点是一样的，可以放心使用。

**Comment by yanglbme** [comment]
已支持
